### PR TITLE
Remove tail() from logging since it's broken anyway

### PR DESCRIPTION
**Pull request by lesnitsky**
Related to #407

### CODE REVIEW

File: resources/googleapis/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 14.0.0
```
Comment (kevmoo) [inline resources/googleapis/CHANGELOG.md:1]:
```suggestion
## 14.0.0-wip
```

same in config. Not ready to publish this yet!

File: config.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # should be ignored.
 packages:
 - googleapis:
-    version: 13.0.0
+    version: 14.0.0
```
Comment (kevmoo) [inline config.yaml:5]:
```suggestion
    version: 14.0.0-wip
```

**Review by kevmoo — CHANGES_REQUESTED** [review]

**Comment by kevmoo** [comment]
The broken test is a known issue with the google auth library...

**Comment by kevmoo** [comment]
I reserve the right to bump this down to a "dot" release. Removing features that don't work is not a big breaking change in my book. 😄

**Comment by lesnitsky** [comment]
Technically it is a breaking change. _Maybe_ there's a library that wraps logging API and uses this method internally, no matter if it doesn't work. `dart pub get` without a lock will break their code :)

Of course, it's up to you to decide whether this is a breaking change. I followed a semver guideline and a method removal is a breaking change.

**Review by kevmoo — APPROVED** [review]
LGTM

**Pull request merged**